modules/skparagraph/include/TextStyle.h

```diff
@@ -259,7 +259,7 @@ class TextStyle {
     void setBaselineShift(SkScalar baselineShift) { fBaselineShift = baselineShift; }
 
     void setHeight(SkScalar height) { fHeight = height; }
-    SkScalar getHeight() const { return fHeightOverride ? fHeight : 0; }
+    SkScalar getHeight() const { return fHeightOverride ? fHeight : 1.0f; }
```

Comment:
Could you explain the logic behind height override, why it was 0.0, and we need 1.0 here?